config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json

```diff
@@ -3,55 +3,36 @@
         "5.1":{
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
+                "release-date": "12 February 2014",
```

Comment:
@GunaKKIBM If I remember correctly, I think we concluded release date as `16 February` based on input from @gayathrimenath-ibm